### PR TITLE
log javascript error in the catch section and stop execution

### PR DESCRIPTION
**Pull request by wkhessairi**

### CODE REVIEW

File: web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js [hunk 1]
```diff
@@ -412,6 +412,8 @@
         eval($("<div />").html(script).text());
         return;
       } catch (err) {
+		  console.error(err.message);
```
Comment (thomasdelhomenie) [inline web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js]: Indentation is not good.

File: web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js [hunk 2]
```diff
@@ -412,6 +412,8 @@
...
+		  return;
```
Comment (thomasdelhomenie) [inline web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js]: We now have a "return" in the try and in the catch, which means the next lines are dead code.

Comment (wkhessairi) [inline web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js]: In fact, the feedback was: when catching an error, next lines should not be executed

Comment (thomasdelhomenie) [inline web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js]: With this PR, the next lines will NEVER be executed. So either they are useless and in this case they have to be removed, or they are useful when an error occurs (catch) and in this case we should not return in the catch.
The initial need is to know what is the error happening in the catch, so adding the log without the return is probably sufficient.

File: web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js [hunk 3]
```diff
@@ -412,6 +412,7 @@
         eval($("<div />").html(script).text());
         return;
       } catch (err) {
+	console.error(err.message);
```
Comment (thomasdelhomenie) [inline web/eXoResources/src/main/webapp/javascript/eXo/portal/PortalHttpRequest.js]: Indentation is not good, should be 2 spaces.

**Pull request closed without merging**